pkg/kubelet/network/dns/dns.go

```diff
@@ -99,6 +100,14 @@ func omitDuplicates(strs []string) []string {
 func (c *Configurer) formDNSSearchFitsLimits(composedSearch []string, pod *v1.Pod) []string {
 	limitsExceeded := false
 
+	for i, search := range composedSearch {
```

Comment:
a comment explaining what we are doing, something like
```
// In glibc 2.28, DNS resolving causes abort() if there is a search path exceeding 255 characters
// so we filter them out
```

Reply:
it is comment in an outdated commit, but I think it applies to the new code :)

Reply:
Thanks. I will add the comment :)